strongbox-security/strongbox-user-management/src/test/java/org/carlspring/strongbox/users/service/AuthorizationConfigFileManagerTest.java

```diff
@@ -0,0 +1,83 @@
+package org.carlspring.strongbox.users.service;
+
+import org.carlspring.strongbox.authorization.AuthorizationConfigFileManager;
+import org.carlspring.strongbox.authorization.dto.AuthorizationConfigDto;
+import org.carlspring.strongbox.authorization.dto.RoleDto;
+import org.carlspring.strongbox.config.DataServiceConfig;
+import org.carlspring.strongbox.config.UsersConfig;
+import org.carlspring.strongbox.users.domain.SystemRole;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+
+import javax.inject.Inject;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * test for {@link AuthorizationConfigFileManager}
+ *
+ * @author Bogdan Sukonnov
+ */
+@SpringBootTest
```

Comment:
@BogdanSukonnov can you please rework this test for not to use the Spring Application Context? The context configuration here is not complicated and can be done manually, this will make our unit tests not overloaded by additional contexts.

Reply:
done